### PR TITLE
Hometask #2

### PR DESCRIPTION
**Pull request by DenysMiroshnyk**

### CODE REVIEW

File: 2/Allergies.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    Arguments:
+        score {integer} -- score of alergies
+
+    Returns:
+        list -- list of things that are person is alergic to
+    """
+    allergens = ["eggs", "peanuts", "shellfish", "strawberries",
+                 "tomatoes", "chocolate", "pollen", "cats"]
+
+    def filter_function(x): return score >> x & 1
```
Comment (oleksandr-medviediev) [inline 2/Allergies.py]: Avoid using one line instructions.

File: 2/Armstrong numbers.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    Returns:
+        bool -- True if is and False otherwise
+    """
+
+    digits_string = str(number)
+    digits_number = len(digits_string)
+    sum = 0
+
+    for digit in digits_string:
+        sum += int(digit)**digits_number
```
Comment (oleksandr-medviediev) [inline 2/Armstrong numbers.py]: spaces around operators!

File: 2/Array difference.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    Arguments:
+        left {list} -- list of integers
+        right {list} -- list of integers subtracted  from the first one
+        
+    Returns:
+        list -- elements of right are subtracted from left
+    """
+    result = left
+
+    for i in result:
```
Comment (oleksandr-medviediev) [inline 2/Array difference.py]: There's a better way to do this.

File: 2/Run length encoding.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+        input_str {str} -- string to encode like "AWWWBBBBA"
+
+    Returns:
+        result {str} -- encoded string like "A3W4BA"
+    """
+    privios_char = input_str[0]
+    semi_char_num = 0
+    result = ""
+
+    for i in input_str:
```
Comment (oleksandr-medviediev) [inline 2/Run length encoding.py]: There's a better way to do this.

File: 2/Allergies.py (hunk 1 above)
Comment (DenysMiroshnyk) [reply 2/Allergies.py]: I've used lambda( vs code formatted it into function definition o_0

File: 2/Armstrong numbers.py (hunk 2 above)
Comment (DenysMiroshnyk) [reply 2/Armstrong numbers.py]: around **?
ok

**Comment by DenysMiroshnyk** [comment]
For each scope, I've made own folder. What should I do with folder structure?

**Pull request merged**